openex-api/src/main/java/io/openex/migration/V3_3__Delete_fk_users_injects.java

```diff
@@ -0,0 +1,26 @@
+package io.openex.migration;
+
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.springframework.stereotype.Component;
+
+import java.sql.Connection;
+import java.sql.Statement;
+
+@Component
+public class V3_3__Delete_fk_users_injects extends BaseJavaMigration {
+
+    @Override
+    public void migrate(Context context) throws Exception {
+        Connection connection = context.getConnection();
+        Statement select = connection.createStatement();
+        select.execute("""
+              ALTER TABLE injects
+              DROP CONSTRAINT fk_a60839b2e20fc097,
+              ADD CONSTRAINT fk_user_id
```

Comment:
Maybe better to have this name : fk_injects_user_id

**Foreign key** on **injects** table on **user_id** property
What do you think ?

Reply:
https://github.com/OpenEx-Platform/openex/pull/293/files#r1335785119